### PR TITLE
Refs [TASK][35836] Init package,base project apply MVP, splash screen 

### PR DESCRIPTION
**Pull request by datvt-2504**
## Related Tickets
- [#Task 35836: Init package,base project apply MVP, splash screen](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- 
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

### ***(pull request task#35836 second time)***

### CODE REVIEW

File: app/src/main/res/layout/tool_bar_screen_child.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        android:background="?android:selectableItemBackground"
+        android:backgroundTint="@android:color/transparent"
+        android:contentDescription="@null"
+        android:src="@drawable/ic_back"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/tool_bar_screen_child.xml]:
```suggestion
        android:id="@+id/textAppName"
```

File: app/src/main/res/layout/tool_bar_screen_parent.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="?actionBarSize"
+    android:background="@color/color_mountain_meadow">
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/tool_bar_screen_parent.xml]: đặt tên vào em

File: app/src/main/java/com/example/dmeal/utils/FragmentUtil.kt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        layoutId: Int,
+        fragment: Fragment,
+        tag: String? = null,
+        addToBackStack: Boolean = true
+    ) {
+        fragmentManager.beginTransaction()
+            .apply {
+                setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
+                add(layoutId, fragment, tag)
+                if (addToBackStack) {
```
Comment (tuannm-1276) [inline app/src/main/java/com/example/dmeal/utils/FragmentUtil.kt]: có cần dòng này ko nhỉ?

Comment (datvt-2504) [reply app/src/main/java/com/example/dmeal/utils/FragmentUtil.kt]: thực ra cái này không cần cũng được ạ

**Pull request merged**